### ISSUE TITLE
Current issues summary

### ISSUE THREAD

**Issue by jm-willy** [jm-willy/Stockcheese#1]
- Gradient is cut by concat -> use concat inside call method of layer, maybe add a trainable tensor in build
- step rewards are too large -> normalize and divide by 100
- training is too slow
- **keras models are instantiated when imported.**. They need to be instantiated multiple times to be different objects.  Use keras.Model
- Use ConvLSTM2Ds 5x5, 3x3, -> LSTM
